cypress/page-objects/loginAngular.js

```diff
@@ -0,0 +1,12 @@
+
+import { VAR } from '../const/constants';
+
+export const IDs = {
+  idSingIn: 'a[href*="/login"]',
+  email: 'input[formcontrolname*="email"]',
+  password: 'input[formcontrolname*="password"]',
+  btnSingIn: 'button[type*="submit"]',
+  errorRegister: 'error-messages',
+  nameProfile: 'a[href*="/profile/prueba%20name"]',
```

Comment:
Could we find another selector? this could change very soon. :/

Reply:
Change in the branch "NewArticule"